actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -46,6 +46,22 @@ def render
       options = @options.stringify_keys
       add_default_name_and_id(options)
       options["input"] ||= dom_id(object, [options["id"], :trix_input].compact.join("_")) if object
+
+      translate_label = options.delete("label")
```

Comment:
I think "Accessible by default" is a good direction for Rails. What do you think about defaulting to `true` for new applications with a configuration along the lines of `config.action_text.generate_aria_label_by_default`. This line would become something like:

```suggestion
      translate_label = generate_aria_label_by_default || options.delete("label")
```